src/TMLeague/wwwroot/data/games/_250120.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Games should have normal names. `250120.json` instead of: `_250120.json`

Reply:
Corrected in next commit